tests/snapshots/files__loop_with_mul_by_inv-optimize-sequential.snap

```diff
@@ -11,20 +11,18 @@ expression: visualization.result
   v5_: int = id c2_;
   v6_: int = id c3_;
   v7_: int = id v0;
-  v8_: int = id c2_;
-.b9_:
-  v10_: int = add v4_ v8_;
+.b8_:
+  v9_: int = mul v5_ v6_;
```

Comment:
This seems like a regression? There used to be no mul in the optimized one afaict
Not sure which optimization does this transformation

Reply:
Oh interesting... I run `cargo run --release tests/passing/small/loop_with_mul_by_inv.bril` locally and it generates the old instructions which does not have `mul`. Must be nondeterminism + greediness of our extractor.


Bril code generated locally:
```
# ARGS: 4
@main(v0: int) {
.b1_:
  c2_: int = const 0;
  c3_: int = const 5;
  v4_: int = id c2_;
  v5_: int = id c2_;
  v6_: int = id c3_;
  v7_: int = id v0;
  v8_: int = id c2_;
.b9_:
  v10_: int = add v4_ v8_;
  c11_: int = const 1;
  v12_: int = add c11_ v5_;
  v13_: int = add v6_ v8_;
  v14_: bool = lt v5_ v7_;
  v4_: int = id v10_;
  v5_: int = id v12_;
  v6_: int = id v6_;
  v7_: int = id v7_;
  v8_: int = id v13_;
  br v14_ .b9_ .b15_;
.b15_:
  print v4_;
  ret;
}
```